spec/lib/middleware/websocket/interactors/client_creation_payload_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe Websocket::Interactor::ClientCreationPayload do
+  let(:client_1) do
+    Websocket::Client.new(connection: double('connection').as_null_object)
+  end
+  let(:client_2) do
+    Websocket::Client.new(connection: double('connection').as_null_object)
+  end
+  let(:clients) { [client_1, client_2] }
+
+  describe '#call' do
+    subject { described_class.new.call(client: client_1) }
+
+    before do
+      allow(client_1.player).to receive(:id)
+      allow(client_1.player).to receive(:token)
+    end
+
+    it 'builds a json payload' do
+      expect(subject).to eq('{"token":null,"id":null}')
```

Comment:
Is this really the output we want?

Reply:
@jgayfer see [this commit](https://github.com/daelynj/typinggame-server/pull/28/commits/acab9e3831b728727b7aa1d43ff0086b407c7215)

is this better?

Reply:
It's technically better, but I don't see a need to do any stubbing here. We can build all of this out from our input.

Reply:
@jgayfer I see what you mean, you were right. I forgot that the `Player` entity isn't automatically generated on initialization of the `Client` object, we actually have to generate it.